Updates to Omeka Bootstrap Theme

BrendonKoz opened the pull request:
`collections/browse.php`:
- Updated sort-links to more closely match Bootstrap styling
- When no associated image for a collection, now displays the
`no-file.png` replacement image
- Updated alignment of description and link to collection so it
left-aligns with collection title instead of wraps below or aligns
right, respectively

`collections/show.php`:
- Updated old bootstrap class of `img-polaroid` with bootstrap3 class of
`img-thumbnail`

`common/breadcrumb.php`:
- Updated breadcrumbs to provide a more correct breadcrumb trail
depending on content and settings
- Added Exhibit Builder `show-item` breadcrumb capability -- full
breadcrumb to first matching Exhibit related to currently viewed item
(if item belongs to multiple exhibits it just uses the first returned
Exhibit from the database since we can't know, via PHP, which Exhibit
the user came from)

`common/header.php`:
- Removed extra orphaned `<div>`

`common/show-selected-metadata.php`:
- Added this file for the `items/show.php` rendering, file seemed to be
missing from repo

`css/style.css`:
- Added helpful classes for compatibility with OpenSeaDragon and Zoomit
Omeka plugins
- Added custom theme styling for `#sort-links` due to change in
`collections/browse.php` to use Bootstrap styling of sort links
- Changed `#search-filters li` styling to better match other theme
content styles

`custom.php`:
- Added optional parameter (and functionality with the parameter) to
`bootstrap_breadcrumb_exhibit_page()` function to allow for the last
element in the breadcrumb trail to also be hyperlinked (used for Exhibit
Item Show page)
- Added `custom_files_for_item()` and `custom_file_markup()` to allow
images to be displayed with HTML5 `<figure>` element markup and
associated filename
- Added `related_exhibit_page()` function as helper to item-show
breadcrumb creation

`items/browse.php`:
- Updated sort-links to more closely match Bootstrap styling
- Updated old bootstrap class of `img-polaroid` with bootstrap3 class of
`img-thumbnail`

`items/show.php`:
- Added `form-group` class to DIV as a hackish fix for better style
matching in two places
- Changed "Files" header to "File" OR "Files" depending on number of
matching items; implemented custom_files_for_item function from updated
theme's `custom.php`

`items/single.php`:
- When no associated image for a collection, now displays the
`no-file.png` replacement image

Commit:
Updates to Omeka Bootstrap Theme

`collections/browse.php`:
- Updated sort-links to more closely match Bootstrap styling
- When no associated image for a collection, now displays the
`no-file.png` replacement image
- Updated alignment of description and link to collection so it
left-aligns with collection title instead of wraps below or aligns
right, respectively

`collections/show.php`:
- Updated old bootstrap class of `img-polaroid` with bootstrap3 class of
`img-thumbnail`

`common/breadcrumb.php`:
- Updated breadcrumbs to provide a more correct breadcrumb trail
depending on content and settings
- Added Exhibit Builder `show-item` breadcrumb capability -- full
breadcrumb to first matching Exhibit related to currently viewed item
(if item belongs to multiple exhibits it just uses the first returned
Exhibit from the database since we can't know, via PHP, which Exhibit
the user came from)

`common/header.php`:
- Removed extra orphaned `<div>`

`common/show-selected-metadata.php`:
- Added this file for the `items/show.php` rendering, file seemed to be
missing from repo

`css/style.css`:
- Added helpful classes for compatibility with OpenSeaDragon and Zoomit
Omeka plugins
- Added custom theme styling for `#sort-links` due to change in
`collections/browse.php` to use Bootstrap styling of sort links
- Changed `#search-filters li` styling to better match other theme
content styles

`custom.php`:
- Added optional parameter (and functionality with the parameter) to
`bootstrap_breadcrumb_exhibit_page()` function to allow for the last
element in the breadcrumb trail to also be hyperlinked (used for Exhibit
Item Show page)
- Added `custom_files_for_item()` and `custom_file_markup()` to allow
images to be displayed with HTML5 `<figure>` element markup and
associated filename
- Added `related_exhibit_page()` function as helper to item-show
breadcrumb creation

`items/browse.php`:
- Updated sort-links to more closely match Bootstrap styling
- Updated old bootstrap class of `img-polaroid` with bootstrap3 class of
`img-thumbnail`

`items/show.php`:
- Added `form-group` class to DIV as a hackish fix for better style
matching in two places
- Changed "Files" header to "File" OR "Files" depending on number of
matching items; implemented custom_files_for_item function from updated
theme's `custom.php`

`items/single.php`:
- When no associated image for a collection, now displays the
`no-file.png` replacement image

BrendonKoz commented:
It seems the orphaned `<div>` is not _always_ orphaned. At least the homepage (index.php) uses it. I might need to look into seeing what other pages use it, if any.

Daniel-KM commented:
Thanks! This is a lot of work. I'll merge it in some days, I have to check some points with my customized themes.

Commit:
Fixing two new caught issues

`collections/show.php`:
- Closing DIV placed outside of if/else block could cause HTML
validation error (orphaned `</div>`)

`css/style.css`:
- Fixes potential layout issue with placement/arrangement of
popover-form labels for advanced search (labels and related checkboxes
now appear correctly aligned (instead of staggered due to float)

`search/search-form.php`:
- Fixed one spot causing HTML validation errors from duplicate IDs
(there are likely more)

BrendonKoz commented:
Figured out the orphaned DIV issue. Added two additional updates. At this point I'll likely be customizing for our own usage (as you have), so that'll probably be the last of the fixes/updates. Thanks for providing such a thorough theme!

Daniel-KM commented:
Thanks for fixes!

BrendonKoz commented:
Thanks for all your work and fixes to gsbodine's version! :)
